src/Microsoft.TestPlatform.ObjectModel/Adapter/TestPlatformFormatException.cs

```diff
@@ -4,11 +4,9 @@
 namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter
 {
     using System;
-    using System.Runtime.Serialization;
 #if NET451
-using System.Security.Permissions;
+    using System.Runtime.Serialization;
```

Comment:
weird change, the namespace that was outside of the compiler directive was moved inside, please revert.

Reply:
reverted